frontend/src/main/js/components/design/scheduleControls.js

```diff
@@ -109,11 +116,46 @@ class ScheduleControls extends Component {
         this.props.controlsStore.setParamsForConnection({ schedule: { locked: false } });
     }
 
+    createCustomParser = e => {
```

Comment:
we can probably not do special parser at all and just check for 'ASAP' in validateStartEnd()